src/main/java/ch/njol/skript/expressions/ExprHoverList.java

```diff
@@ -111,24 +98,57 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 
 	@SuppressWarnings("null")
 	@Override
-	public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
-		if (!(e instanceof PaperServerListPingEvent))
+	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
+		if (!(event instanceof PaperServerListPingEvent))
 			return;
 
+		if (HAS_NEW_LISTED_PLAYER_INFO) {
+			List<PaperServerListPingEvent.ListedPlayerInfo> values = new ArrayList<>();
+			if (mode != ChangeMode.DELETE && mode != ChangeMode.RESET) {
+				for (Object o : delta) {
+					if (o instanceof Player) {
+						Player player = (Player) o;
+						values.add(new PaperServerListPingEvent.ListedPlayerInfo(player.getName(), player.getUniqueId()));
+					} else {
+						values.add(new PaperServerListPingEvent.ListedPlayerInfo((String) o, UUID.randomUUID()));
```

Comment:
Does this have the 15 character limit on names? If so, we need to verify this beforehand to prevent an error.

Reply:
no, i tried with a 30 ish char string and it worked on both versions